### PR TITLE
Update gov-params.md

### PR DESCRIPTION
**Pull request by dgsbl**

### CODE REVIEW

File: docs/concepts/gov-params.md [hunk 1]
```diff
@@ -5,74 +5,126 @@ All the IRIS holders may participate in the on-chain governance. If the communit
...
+
+| key                 | Description                                      | Range                                                    | Current                                                      |
+| ------------------- | ------------------------------------------------ | -------------------------------------------------------- | ------------------------------------------------------------ |
+| `gov/depositparams` | Related parameters of the deposit mortgage phase | max_deposit_period:(0, +∞)                               | {"min_deposit": [{"denom": "uiris", "amount": "1000000000"}], "max_deposit_period": "86400s" }, |
+| `gov/votingparams`  | Related parameters of the voting mortgage phase  | voting_period:(0, +∞)                                    | {"voting_period": "432000s"}                                 |
+| `gov/tallyparams`   | Related parameters of the voting tally phase     | quorum:[0,1]<br>threshold:(0,1]<br/>veto_threshold:(0,1] | {"quorum":"0.500000000000000000","threshold": "0.500000000000000000","veto_threshold": "0.330000000000000000"} |
+
+Details in [Governance](../features/governance.md)
+
+## Parameters in ibc
```
Comment (chengwenxi) [inline docs/concepts/gov-params.md:60]:
```suggestion
## Parameters in IBC
```

**Review by chengwenxi — COMMENTED** [review]
Could you rearrange the order of modules?

File: docs/concepts/gov-params.md [hunk 2]
```diff
@@ -5,74 +5,126 @@ All the IRIS holders may participate in the on-chain governance. If the communit
...
+| `service/ComplaintRetrospect`  | Maximum time for submitting a dispute                       | (0, 9223372036854775807]  | 360h0m0s |
+| `service/MaxRequestTimeout`    | Maximum number of blocks to wait for service invocation     | (0, 9223372036854775807]  | 100      |
+| `service/MinDepositMultiple`   | A multiple of the minimum deposit amount of service binding | (0, 9223372036854775807]  | 1000     |
+| `service/ServiceFeeTax`        | Tax rate of service fee                                     | [0, 1)                    | 0.01     |
+| `service/SlashFraction`        | Slash fraction                                              | [0, 1]                    | 0.001    |
+| `service/TxSizeLimit`          | The limit of the service tx size                            | (0, 18446744073709551615] | 4000     |
+| `service/MinDeposit`           | Minimum deposit amount                                      | amount: (0, +∞)           | [{"denom": "uiris","amount": "5000000000"}] |
+| `service/BaseDenom`            | Tokens supported by service fees                            |                           | uiris    |
```
Comment (haifengxi) [inline docs/concepts/gov-params.md:87]: BaseDenom: Token denom that must be used for deposits

**Review by haifengxi — CHANGES_REQUESTED** [review]

File: docs/zh/concepts/gov-params.md [hunk 3]
```diff
@@ -1,78 +1,130 @@
...
-| 字段                     | 描述             | 有效范围            | 当前值        |
-| ------------------------ | ---------------- | ------------------- | ------------- |
-| `auth/gasPriceThreshold` | 最小的gas单价    | (0, 10^6uiris] | 6000000000000 |
-| `auth/txSizeLimit`       | 普通交易大小限制 | [500, 1500]         | 1000          |
+| 字段                           | 描述                              | 有效范围                   | 当前值   |
+| ----------------------------- | -------------------------------- | ------------------------- | ------- |
+| `auth/MaxMemoCharacters`      | 交易的meno字段的最大字符数           | (0, 18446744073709551615] | 256     |
+| `auth/TxSigLimit`             | 每个交易的最大签名数                 | (0, 18446744073709551615] | 7       |
+| `auth/TxSizeCostPerByte`      | 交易每个字节消耗的gas量              | (0, 18446744073709551615] | 10      |
+| `auth/SigVerifyCostED25519`   | 在EDD2519算法签名验证上花费的gas     | (0, 18446744073709551615] | 590     |
```
[the remark this answers is not in the document]
Comment (secret2830) [reply docs/zh/concepts/gov-params.md:13]: 交易的memo字段的最大字符数

**Review by chengwenxi — CHANGES_REQUESTED** [review]
Please check zh params also

**Review by chengwenxi — APPROVED** [review]

**Review by SegueII — APPROVED** [review]

**Review by dreamer-zq — APPROVED** [review]

**Review by haifengxi — APPROVED** [review]

**Pull request merged**